packages/neutrino-preset-react/index.js

```diff
@@ -39,8 +40,19 @@ module.exports = (neutrino) => {
       'react/lib/ReactContext': 'window'
     })
     .when(process.env.NODE_ENV === 'development', config => config
+      .module
```

Comment:
I suppose it should be done only in the "development" and should not increase the bundle size without a necessary